packages/vincent-tools/vincent-tool-uniswap-swap/src/generated/vincent-tool-ipfs-cid.js

```diff
@@ -0,0 +1,5 @@
+const vincentToolIpfsCid = 'QmaMhVwwC6T5P3pN4SxRNfmN62RCETy4o7dkh5XGbZcrWa';
```

Comment:
As mentioned above, we need the ipfsCid to be stored as json and imported into a .ts file that we then transpile to `.js` using our standard build (e.g. tsc will likely just do it for us)